Formula/node@12.rb

```diff
@@ -0,0 +1,47 @@
+class NodeAT12 < Formula
+  desc "Platform built on V8 to build network applications"
+  homepage "https://nodejs.org/"
+  url "https://nodejs.org/dist/v12.13.0/node-v12.13.0.tar.gz"
+  sha256 "2e5321e095fe673a3ab936cf77faf8c983cba62f27a9fbd00530a7edb739a040"
+
+  keg_only :versioned_formula
+
+  depends_on "pkg-config" => :build
+  depends_on "python@2" => :build # does not support Python 3
+  depends_on "icu4c"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--with-intl=system-icu"
+    system "make", "install"
+  end
+
+  def post_install
+    (lib/"node_modules/npm/npmrc").atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+  end
+
+  test do
+    path = testpath/"test.js"
+    path.write "console.log('hello');"
+
+    output = shell_output("#{bin}/node #{path}").strip
+    assert_equal "hello", output
+    output = shell_output("#{bin}/node -e 'console.log(new Intl.NumberFormat(\"en-EN\").format(1234.56))'").strip
+    assert_equal "1,234.56", output
+
+    output = shell_output("#{bin}/node -e 'console.log(new Intl.NumberFormat(\"de-DE\").format(1234.56))'").strip
+    assert_equal "1.234,56", output
+
+    # make sure npm can find node
+    ENV.prepend_path "PATH", opt_bin
+    ENV.delete "NVM_NODEJS_ORG_MIRROR"
+    assert_equal which("node"), opt_bin/"node"
+    assert_predicate HOMEBREW_PREFIX/"bin/npm", :exist?, "npm must exist"
```

Comment:
The test for `node@12` has to be changed to assume `npm` in `bin/"npm"` and not in `HOMEBREW_PREFIX/"bin/npm"`:

```suggestion
    assert_predicate bin/"npm", :exist?, "npm must exi
```

and so on, see also the test from `node@10`:
https://github.com/Homebrew/homebrew-core/blob/b2032988779682d1fcfed259952f7e1628dfdf89/Formula/node%4010.rb#L47-L58

Reply:
The test for `node@12` has to be changed to assume `npm` in `bin/"npm"` and not in `HOMEBREW_PREFIX/"bin/npm"`:

```suggestion
    assert_predicate bin/"npm", :exist?, "npm must exist"
```

and so on, see also the test from `node@10`:
https://github.com/Homebrew/homebrew-core/blob/b2032988779682d1fcfed259952f7e1628dfdf89/Formula/node%4010.rb#L47-L58